LW_README.md

```diff
@@ -0,0 +1,36 @@
+# Apache Spark for Lacework
+
+Main reasons for lacework to fork Apache spark source code -
+1. Spark current version is building source code with deprecated `openjdk:11-jre-slim`( refer this URL for notice - https://hub.docker.com/_/openjdk), this means build is older, missing any security patches or bug fixes.
+2. Lacework batch/service code is built using Amazon Corretto Java 8 but has problem running on `openjdk:11-jre-slim`.
```

Comment:
Minor nit: Since this is a public repo (and probably has to be for technical reasons), let's not mention details of Lacework's infrastructure. 

Same on Line 9.